lattice-control/lattice-control-interface.smithy

```diff
@@ -59,25 +60,27 @@ operation AuctionActor {
 /// a "gather" operation and so can be influenced by short timeouts,
 /// network partition events, etc.
 operation GetHosts {
+    input: GetHostsRequest
```

Comment:
Could we simply take the lattice ID instead of the GetHostsRequests here? That would make the usage a little cleaner